avocado-setup.py

```diff
@@ -487,25 +488,28 @@ def parse_test_config(test_config_file, avocado_bin, enable_kvm):
                 logger.debug("%s does not exist", test_dic['test'])
                 continue
             if len(line) > 1:
-                test_dic['mux'] = line[1]
-                mux_flag = 1
-                test_dic['name'] = "%s_%s" % (test_dic['name'], test_dic['mux'].split("/")[-1].split(".")[0])
-                if args.inputfile:
-                    mux_file = os.path.join(TEST_DIR, test_dic['mux'])
-                    if not os.path.isfile(mux_file):
-                        logger.debug("%s does not exist", mux_file)
-                        continue
-                    tmp_mux_path = os.path.join('/tmp/mux/', "%s_%s.yaml" % (test_config_name, test_dic['name']))
-                    edit_mux_file(test_config_name, mux_file, tmp_mux_path)
-                    test_dic['mux'] = tmp_mux_path
+                if '.yaml' in line[1]:
+                    test_dic['mux'] = line[1]
+                    mux_flag = 1
+                    test_dic['name'] = "%s_%s" % (test_dic['name'], test_dic['mux'].split("/")[-1].split(".")[0])
+                    if args.inputfile:
+                        mux_file = os.path.join(TEST_DIR, test_dic['mux'])
+                        if not os.path.isfile(mux_file):
+                            logger.debug("%s does not exist", mux_file)
+                            continue
+                        tmp_mux_path = os.path.join('/tmp/mux/', "%s_%s.yaml" % (test_config_name, test_dic['name']))
+                        edit_mux_file(test_config_name, mux_file, tmp_mux_path)
+                        test_dic['mux'] = tmp_mux_path
+                else:
+                    test_dic['args'] = " %s" % line[1]
             count = 0
             for list_dic in test_list:
                 if test_dic['name'] == list_dic['name'].split('.')[0]:
                     count += 1
             if count:
                 test_dic['name'] += ".%d" % (count + 1)
             if len(line) > 2:
-                test_dic['args'] = " --execution-order %s " % line[2]
+                test_dic['args'] = " %s" % line[2]
             test_list.append(test_dic)
         if mux_flag == 0:
```

Comment:
@harish-24 `if mux_flag == 0 and not test_dic['args']:`
Even mux check can be similar to args in the above

Reply:
@narasimhan-v Oh, Thanks for pointing out. Will change and revert